### PR TITLE
User post

### PR DESCRIPTION
**Pull request by Harih17**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,87 +1,86 @@
...
+            <artifactId>mapstruct</artifactId>
+            <version>${mapstruct.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.mapstruct</groupId>
+            <artifactId>mapstruct-processor</artifactId>
+            <version>${mapstruct.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.h2database</groupId>
```
Comment (shreysmh) [inline pom.xml]: remove this dependency, this will be taken care in another PR

File: pom.xml [hunk 2]
```diff
@@ -1,87 +1,86 @@
...
+            <scope>runtime</scope>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-validation</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
```
Comment (shreysmh) [inline pom.xml]: Remove this as well, as it is part of another development

File: pom.xml (hunk 1 above)
Comment (Harih17) [reply pom.xml]: removed dependency

File: pom.xml (hunk 2 above)
Comment (Harih17) [reply pom.xml]: removed dependency

**Review by shreysmh — APPROVED** [review]
